graphiql/graphiql.go

```diff
@@ -0,0 +1,167 @@
+package graphiql
+
+import "github.com/gramework/gramework"
+
+const (
+	// PackageVersion defines graphiql package version
+	// from npm, which we'll use in our unpkg.com links
+	PackageVersion = "0.11.11"
+)
+
+// Handler serves standalone graphiql HTML.
+func Handler(ctx *gramework.Context) {
+	ctx.HTML().WriteString(tpl)
```

Comment:
Error return value of `(*github.com/gramework/gramework/vendor/github.com/valyala/fasthttp.RequestCtx).WriteString` is not checked